spec/expression.dd

```diff
@@ -1907,6 +1907,18 @@ $(GNAME ValueExpression):
         are inserted as arguments in place of the sequence.
     )
 
+    $(P You can declare duplicate keys in the $(I KeyValuePairs). The
+        value returned for that key is going to be that of the last
+        lexicographically $(I KeyValuePair) declared. This is because
+        duplicates are not actually stored, the value for that key is
+        just updated to the last assigned value.
+    )
```

Comment:
The message is fine, but the wording is not quite adequate for technical prose. Avoid using "you" and try to use as much as possible the impersonal 3rd person:

"Associative array initializers may contain duplicate keys, however,  in that case, the last lexicographically $(I KeyValuePair) encountered is stored".

Much more concise and technical.

Reply:
Thanks for the advice!